web/headref_display.go

```diff
@@ -41,6 +45,10 @@ func (web *Web) headrefDisplayHandler(w http.ResponseWriter, r *http.Request) {
 
 // The websocket endpoint for the ref display client to receive status updates.
 func (web *Web) headrefDisplayWebsocketHandler(w http.ResponseWriter, r *http.Request) {
+	if !web.userIsExpected(w, r, []string{adminUser, refUser}) {
```

Comment:
should there be some sort of error message returned to the client here? something equivalent to `handleWebErr`